src/Help.js

```diff
@@ -12,13 +12,12 @@ import { spacers, theme } from './theme.js'
  * @example import { Help } from @dhis2/ui-core
  * @see Live demo: {@link /demo/?path=/story/help--default|Storybook}
  */
-const Help = ({ children, valid, error, warning, className, indent }) => (
+const Help = ({ children, valid, error, warning, className }) => (
```

Comment:
The indentation for the `Help` was mainly for the `filled`, so let's get rid of it.